src/components/localization/localized-link.js

```diff
@@ -65,7 +65,14 @@ export const LocalizedLink = ({ to, ...props }) => {
 
     if (props.external || props.external === 'true') {
         return (
-            <a target={target} rel={rel} className={className} style={style} href={to}>
+            <a
+                target={target}
+                rel={rel}
+                className={className}
+                data-amp-replace="QUERY_PARAM"
```

Comment:
what is `data-amp-replace`?

Reply:
this is a built in amp function, that will redirect query parameter to non amp page, amp page cant access window, localstorage, cookies, etc. so this is the only option to redirect the query parameter

Reply:
oh I see, thanks for clarifying @mustofa-binary